package.json

```diff
@@ -34,9 +34,13 @@
   },
   "dependencies": {
     "async": "^2.1.5",
-    "gulp-util": "^3.0.8",
+    "chalk": "^2.3.0",
+    "fancy-log": "^1.3.2",
     "lodash": "^4.17.4",
-    "through2": "^2.0.3"
+    "lodash.template": "^4.4.0",
+    "plugin-error": "^0.1.2",
+    "through2": "^2.0.3",
+    "vinyl": "^2.1.0"
```

Comment:
Vinyl should be in the `devDependencies`. It is only used for the tests.
Please update it: it is important to avoid unnecessary dependencies.

Reply:
Oops, that's true. I created another PR for that https://github.com/sun-zheng-an/gulp-shell/pull/96